streams/src/main/java/org/apache/kafka/streams/processor/internals/InternalTopologyBuilder.java

```diff
@@ -364,6 +371,10 @@ public synchronized final StreamsConfig getStreamsConfig() {
         return config;
     }
 
+    public String namedTopology() {
```

Comment:
Ditto, I feel its better to let `topologyName` return a String.

Reply:
You mean to rename this to `topologyName()`? Ack